### DISCUSSION TITLE
Can not activate virtual env for chia to update to 1.1.6

### DISCUSSION THREAD

**Discussion by Towhid999** [Chia-Network/chia-blockchain#5936] [Q&A] [answered]
Everything was running smooth untill i tried to update chia to 1.1.6.
tried upgrading it but its still shows 1.1.5
Then friend of mine suggest to delete the chia-blockchain folder and install fresh.
So, I deleted the chia-blockchain folder and started according to the guide,
sudo apt-get update
sudo apt-get upgrade -y
sudo apt install git -y
git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules
cd chia-blockchain
sh install.sh

I keep trying to install but cant get past . ./activate. it says "bash: ./activate: No such file or directory" 

Here is my ls 
azure-pipelines.yml  CONTRIBUTING.md      mypy.ini
build_scripts        install-gui.sh       pyproject.toml
BUILD_TIMELORD.md    INSTALL.md           README.md
CHANGELOG.md         install.sh           run-py-tests.sh
chia                 install-timelord.sh  setup.py
chia-blockchain-gui  LICENSE              tests
CODE_OF_CONDUCT.md   mozilla-ca

My python3 --version is 3.8.5

sh install.sh command gives me this--

pt-get is /usr/bin/apt-get
Installing on Ubuntu 20.04 LTS.
[sudo] password for towhid: 
Ign:1 http://ppa.launchpad.net/indicator-brightness/ppa/ubuntu focal InRelease
Hit:2 http://bd.archive.ubuntu.com/ubuntu focal InRelease                    
Get:3 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]  
Get:4 http://bd.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]   
Err:5 http://ppa.launchpad.net/indicator-brightness/ppa/ubuntu focal Release 
  404  Not Found [IP: 91.189.95.85 80]
Get:6 http://bd.archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB] 
Reading package lists... Done     
E: The repository 'http://ppa.launchpad.net/indicator-brightness/ppa/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

Now I dont know what to do. Please help to update to 1.1.6

**Comment by rcamarda390** [comment]
you deleted the chia-blockchain directory, but what about the .chia directory?
maybe chia got confused

**Reply by Towhid999** [reply]
you are right, possibly.
I deleted the chia-blockchain directory. 
Didnt delete the .chia directory. 
I didn't delete it because I thought it would delete all the sync node info, which will take hours to resync again...
should I delete it too and try again?
thanks

**Reply by lvcivs** [reply]
No, that's not the issue here. See my answer.

**Comment by lvcivs** [comment, marked as answer]
`apt` can't finde one of the PPAs that you have configured as third party repositories: the indicator-brigthness ppa. So it aborts and the installation of chia does not complete.

You need to clean up your third party repositories first.

**Reply by Towhid999** [reply]
Thank you brother from another mother. love you.
